docs/index.rst

```diff
@@ -72,18 +72,31 @@ The best way to get started with Parcels is to have a look at the Jupyter notebo
 Installing Parcels
 ===================================
 
-The latest version of Parcels, including tests and examples, 
-can be obtained directly from github via::
-    git clone https://github.com/OceanParcels/parcels.git
-    cd parcels; pip install -r requirements.txt
-    python scripts/pull_data.py
-    export PYTHONPATH="$PYTHONPATH:$PWD"
-
-In order for Parcels to work from any directory, add the following line to 
-your ~/.bash_profile::
-    export PYTHONPATH="$PYTHONPATH:$PWD"
-
-Note that a functional NetCDF install is required.
+Parcels depends on a working Python installation, a netCDF installation, a C
+compiler, and various Python packages.  These requirements can be met in
+different ways:
+
+1. Use Anaconda and Conda-Forge for all requirements.  This is the
+   “batteries-included” solution probably suitable for most users.
+
+2. Use an existing netCDF installation and an existing C compiler and just get
+   the Python part of the dependencies.  This may be a suitable solution for
+   more experienced users who want to control the details.
+
+Installing Parcels and all dependencies with Anaconda and Conda-Forge
+-----------------------------------
+
+Install Anaconda's Miniconda following the steps at https://conda.io/docs/user-guide/install/
+
+Download the `environment.yml <https://raw.githubusercontent.com/OceanParcels/parcels/install_via_conda/environment.yml>`_ file and place it in the directory where you want to install Parcels. Then, run::
+
+    source $HOME/miniconda/bin/activate root
+    conda env create -f environment.yml
```

Comment:
Another thing:  The installation is only completed after using pip to install from the `master` branch of Parcels.  So the whole code block is:
```bash
source $HOME/miniconda/bin/activate root
conda env create -f environment.yml -n py2_parcels
pip install git+https://github.com/OceanParcels/parcels.git@master
```